sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/implementation/EventHubConnectionProcessor.java

```diff
@@ -0,0 +1,268 @@
+/*
+ * Copyright (c) 2011-2017 Pivotal Software Inc, All Rights Reserved.
```

Comment:
This is not the copyright text we use. Is `EventHubConnectionProcessor` code provided by Pivotal Software and they require us to have this copyright text instead?

Reply:
Oh no. I have no idea where I had that from.